packages/lib/src/interactions/DefaultInteractions.tsx

```diff
@@ -0,0 +1,51 @@
+import { merge } from 'lodash';
+
+import Pan from '../interactions/Pan';
+import SelectToZoom from '../interactions/SelectToZoom';
+import XAxisZoom from '../interactions/XAxisZoom';
+import YAxisZoom from '../interactions/YAxisZoom';
+import Zoom from '../interactions/Zoom';
+import type { Interactions } from './models';
+import { getDefaultInteractions } from './utils';
+
+interface Props {
+  interactions?: Interactions;
+  keepRatio?: boolean;
+}
+
+function DefaultInteractions(props: Props) {
+  const { interactions: givenInteractions = {}, keepRatio } = props;
+
+  const parsedInteractions = Object.fromEntries(
+    Object.entries(givenInteractions).map(([k, v]) => {
+      if (v === true) {
+        return [k, {}];
+      }
+
+      if (v === false) {
+        return [k, null];
```

Comment:
I put `null` instead of `undefined` as fields with `undefined` will not overwrite the default value:

> Source properties that resolve to undefined are skipped if a destination value exists.
>
> https://lodash.com/docs/4.17.15#merge